app/models/episode.rb

```diff
@@ -42,7 +42,7 @@ def self.by_prx_story(story)
   end
 
   def publish_updated
-    podcast.publish_updated if podcast
+    podcast.publish_updated if podcast && published?
```

Comment:
I sort of think you should always update the podcast here.  What if the update to this episode was that it was unpublished?